logger.js

```diff
@@ -79,7 +79,7 @@ logger.dump_and_exit = function (code) {
 
 logger.log = function (level, data) {
     if (level === 'PROTOCOL') {
-        data = data.replace(/\n/g, '\\n\n');
+        data = data.replace(/\n/g, '\\n');
```

Comment:
Exactly. When else would there ever be a `\n\n` in the line? (possible answer: wrapped headers, but then, don't you want to strip them anyway and have you're really long lines that your editor handles?)